File: Software Development Module/Week2/AreaRectangle.java
/*
AreaRectangle.java
Billy Cussen
07/07/2020
*/

//This class will ask the user for the length and width of a Rectangle and produce its Area

import java.util.Scanner;

public class AreaRectangle{

    public static void main(String [] args){

        Scanner keyboard = new Scanner(System.in);
        System.out.print("Enter the length of the Rectangle: ");
        double length = keyboard.nextDouble();

        System.out.print("Enter the width of the Rectangle: ");
        double width = keyboard.nextDouble();
        keyboard = null;

        double area = length*width;

        System.out.println("The area of the Rectangle is: "+area);
    }

}

File: Python/Software Development Module - Python/Week2/AreaRectangle.py
"""
AreaRectangle.py
Billy Cussen
23/09/2020
"""

repeat = "y"
inputCheck = False

while repeat == "y":

    while inputCheck == False:
        try:
            length = float(input("Please enter the Rectangles Length: "))
        except:
            print("You need to enter a Number!")
        else:
            if length<=0:
                print("The length of the rectangle needs to be 1 or greater!")
            else:
                inputCheck = True

    while inputCheck == True:
        try:
            width = float(input("Please enter the Rectangles Width: "))
        except: 
            print("You need to enter a number!")
        else:
            if width<=0:
                print("The Rectangles width needs to be 1 or greater")
            else:
                print("The area of the Rectangle is: "+str(length*width))
                inputCheck = False;

    repeat = ""

    while repeat != "y" and repeat != "n":
        repeat = input("Please enter if you would like to retry (\"Y\" or \"N\"): ")
        repeat = repeat.lower()

        if(repeat!="y" and repeat != "n"):
            print("You need to enter Y or N")

print("Thanks for using my App!")    